### ISSUE TITLE
not able to login from vs code even after successful login from browser

### ISSUE THREAD

**Issue by mohit-kaul-kr** [microsoft/vscode-azure-account#860]
Does this occur consistently? 
Repro steps:

1.
2.

Version: 0.11.5
OS: win32
OS Release: 10.0.19045
Product: Visual Studio Code
Product Version: 1.82.2
Language: en

**Comment by alexweininger** [comment]
Please add detailed steps to reproduce the issue.

Are you behind a proxy of any kind? Proxies often cause issues with sign-in. See the [Unable to Sign In While Using a Proxy section of our wiki](https://github.com/microsoft/vscode-azure-account/wiki/Troubleshooting#unable-to-sign-in-while-using-a-proxy) for possible fixes.

Also, see the [Enable and View Logs](https://github.com/microsoft/vscode-azure-account/wiki/Enable-and-view-logs) section of the wiki for how to capture and share Azure Account extension logs.

**Comment by mohit-kaul-kr** [comment]
I'm trying to access event hub from vs code, I downloaded azure account extension and also event hub extension, when I try to login to my azure account it redirect me to browser login and after successful login on browser when I go back to vs code it won't happen there, and again it ask me to sign in.

**Comment by mohit-kaul-kr** [comment]
I’m trying to access event hub from vs code, I downloaded azure account extension and also event hub extension, when I try to login to my azure account it redirect me to browser login and after successful login on browser when I go back to vs code it won’t happen there, and again it ask me to sign in.

**Comment by alexweininger** [comment]
Can you provide the Azure Account extension logs?

**Comment by mohit-kaul-kr** [comment]
is there any doc to follow for enabling logs

**Comment by alexweininger** [comment]
See the [Enable and View Logs](https://github.com/microsoft/vscode-azure-account/wiki/Enable-and-view-logs) section of the wiki for how to capture and share Azure Account extension logs.

**Comment by aaronbir** [comment]
@mohit-kaul-kr  try using Edge browser to authenticate. This worked for me

**Comment by alexweininger** [comment]
It seems like something is being cached in your browser so as @aaronbir mentioned, please try with a different browser. Logging in/out or closing and reopening the browser may also help.

**Comment by mohit-kaul-kr** [comment]
i already tried opening with different browser, uninstalling and installing vs code but i'm still getting same error, got login successfully on browser but it's not redirected and not successfully login on vs code

**Comment by mohit-kaul-kr** [comment]
any help on the above issue, i tried clearing cookies but it's still the same issue

**Comment by mohit-kaul-kr** [comment]
here are the logs as requested
[logs.json](https://github.com/microsoft/vscode-azure-account/files/12887195/logs.json)

**Comment by alexweininger** [comment]
Can you try using the "Sign In with Device Code" command? You can access it from the command palette using F1.

**Comment by mohit-kaul-kr** [comment]
i tried using the command palette, it prompted me to do the sign on on the browser but the redirect still not working, still can't login on the vs code

**Comment by alexweininger** [comment]
Did you try using the "Sign In with Device Code"?

**Comment by mohit-kaul-kr** [comment]
yeah sorry i missed typing there that i sign in with device code from command palette and it's still the same issue
